### PR TITLE
Update broken link

### PR DESCRIPTION
**Pull request by rafaelugolini**

### CODE REVIEW

**Review by benface — APPROVED** [review]
🤣

**Comment by dimitrovmaksim** [comment]
:arrow_upper_left: :facepalm:

**Comment by rafaelugolini** [comment]
@benface, since there was a conflict, I rebased and also updated the links in the other languages

**Comment by rafaelugolini** [comment]
> @benface, since there was a conflict, I rebased and also updated the links in the other languages

I just saw that I shouldn't have done so I deleted the previous commit and only left the rebase

**Comment by benface** [comment]
Thank you @rafaelugolini!

> I just saw that I shouldn't have done so I deleted the previous commit and only left the rebase

Actually, for fixing this sort of stuff, it's perfectly fine to do it in all languages – when Crowdin updates those files eventually, surely the same fix will have been applied there too. But it's better if we don't have to wait for that.

**Comment by rafaelugolini** [comment]
@benface I added the fix back :)

File: pages/en/developing/unit-testing-framework.mdx [hunk 1]
```diff
@@ -1072,7 +1072,7 @@ The log output includes the test run duration. Here's an example:
 
 > Critical: Could not create WasmInstance from valid module with context: unknown import: wasi_snapshot_preview1::fd_write has not been defined
 
-This means you have used `console.log` in your code, which is not supported by AssemblyScript. Please consider using the [Logging API](http://localhost:3000/en/developing/assemblyscript-api/#logging-api)
+This means you have used `console.log` in your code, which is not supported by AssemblyScript. Please consider using the [Logging API](https://thegraph.com/docs/en/developer/assemblyscript-api/#logging-api)
```
Comment (benface) [inline pages/en/developing/unit-testing-framework.mdx:1075]:
```suggestion
This means you have used `console.log` in your code, which is not supported by AssemblyScript. Please consider using the [Logging API](/developer/assemblyscript-api/#logging-api)
```

**Review by benface — CHANGES_REQUESTED** [review]
Ugh, I'm sorry for not noticing when I reviewed the first time, but links within content should ideally be (root-) relative and not absolute. That will also ensure that the links don't direct the user to a different language.

**Comment by rafaelugolini** [comment]
@benface fixed :)

**Review by benface — APPROVED** [review]
Thank you so much, and sorry for the back and forth!

**Comment by rafaelugolini** [comment]
> Thank you so much, and sorry for the back and forth!

all good :)

**Pull request merged**